Content.Shared/_Goobstation/Devil/UI/RevivalContractMessage.cs

```diff
@@ -0,0 +1,10 @@
+using Robust.Shared.Player;
+using Robust.Shared.Serialization;
```

Comment:
unused using directive `using Robust.Shared.Player`

```suggestion
using Robust.Shared.Serialization;
```